test/mocha/50-records-api.js

```diff
@@ -18,10 +20,13 @@ let signedConfig;
 describe('Records API', () => {
   before(async function() {
     await helpers.prepareDatabase(mockData);
+    const key =
+      await Ed25519VerificationKey2020.from(mockData.keys.authorized);
```

Comment:
Style here and everywhere else is break on params.

```suggestion
    const key = await Ed25519VerificationKey2020.from(
      mockData.keys.authorized);
```